### PR TITLE
add agility-0.1.3

### PR DESCRIPTION
**Pull request by glensc**
plugin homepage: https://github.com/arturadib/agility

   - Watch 33
   - Star 558
   - Fork 77

----

1. created temporary repo to add `package.json` (https://github.com/glensc/agility)
2. ran cdnjs-importer tool (`./bin/cdnjs-importer -p /home/users/glen/scm/cdnjs/cdnjs -g https://github.com/glensc/agility`), but it [failed](https://github.com/cdnjs/cdnjs-importer/issues/24), so created all changes manually including creating `.min.js`
3. npm test passed (`  ✓ OK » 24397 honored (4.844s)`)

btw, the package does not have npm registry entry, you sure `filename` should be required entry?

```
    agility 
      ✗ agility package.json is valid 
        » agility is not a valid cdnjs package.json format: 
          » NPM package.json 
                Property is required (true): /properties/filename // valid-packages-test.js:82 
   
      ✗ agility: filename from package.json exists 
        » ./ajax/libs/agility/0.1.3/undefined does not exist but is referenced in package.json! // valid-packages-test.js:101 
  ✗ Broken » 24395 honored ∙ 2 broken (15.335s) 
```

### CODE REVIEW

File: ajax/libs/agility/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "agility",
+  "description": "Javascript MVC for the \"write less, do more\"",
+  "homepage": "https://github.com/arturadib/agility",
+  "version": "0.1.3",
+  "licences": [
```
Comment (PeterDaveHello) [inline ajax/libs/agility/package.json]: Please use `licence` instead of `licences` since there is only MIT license here, thanks.

**Comment by PeterDaveHello** [comment]
> btw, the package does not have npm registry entry, you sure filename should be required entry?

filename points to the main file, and has nothing to do with npm, so that's fine.

**Comment by PeterDaveHello** [comment]
@glensc please auto apply auto-update config for it, here is an example for this lib, thanks. 
```js
 "autoupdate": {
    "source": "git",
    "target": "git://github.com/arturadib/agility.git",
    "basePath": "",
    "files": [
      "agility*"
    ]
  }
```

**Comment by glensc** [comment]
added, also squashed and rebased against current master!

**Comment by PeterDaveHello** [comment]
Thanks @glensc

**Pull request merged**